docs/src/test-parallel-js.md

```diff
@@ -3,61 +3,155 @@ id: test-parallel
 title: "Parallelism and sharding"
 ---
 
-Playwright Test runs tests in parallel by default, using multiple worker processes.
+Playwright Test uses multiple worker processes to run tests in parallel. Each worker process creates a new environment to run tests. By default, Playwright Test reuses the worker as much as it can to make testing faster.
```

Comment:
I am not sure this is the language we want to speak to the @playwright/test customers. We just need to say that tests run in parallel, we don't need to talk about the worker processes right away?